storage/range_command.go

```diff
@@ -339,6 +343,8 @@ func (r *Range) InternalRangeLookup(batch engine.Engine, args *proto.InternalRan
 		reply.SetGoError(util.Errorf(
 			"Range lookup specified invalid maximum range count %d: must be > 0", rangeCount))
 		return
+	} else if args.IgnoreIntents {
```

Comment:
Remove the `else`. It's unnecessary after the return in the `if`, and it's harder to follow when the structure implies that `args.IgnoreIntents` and `rangeCount < 1` are somehow related.